From 8aa91d500c0fbd504e9fc8f61e987d6340eb1ad8 Mon Sep 17 00:00:00 2001
From: Marcelo Fernandez <marcelofg55@gmail.com>
Date: Sat, 25 Mar 2017 12:29:54 -0300
Subject: [PATCH] Shared library now outputs its name to liboaml (without the
 _shared suffix)

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 71433c0..a2c229f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -213,6 +213,7 @@ if (ENABLE_SHARED)
 	target_link_libraries(oaml_shared ${OAML_LIBS})
 	set_target_properties(oaml_shared
 		PROPERTIES
+			OUTPUT_NAME oaml
 			SOVERSION ${PROJECT_VERSION_MAJOR}
 			VERSION   ${PROJECT_VERSION}
 	)
